Tools/Gulp/config.json

```diff
@@ -1028,7 +1028,8 @@
                 "../../src/Gizmos/babylon.planeRotationGizmo.js",
                 "../../src/Gizmos/babylon.positionGizmo.js",
                 "../../src/Gizmos/babylon.rotationGizmo.js",
-                "../../src/Gizmos/babylon.scaleGizmo.js"
```

Comment:
I think we should create a workload for gizmos
Also utilitylayer should be independent and gizmos should have a ref on it